packages/mobile/src/exchange/ExchangeConfirmationCard.tsx

```diff
@@ -130,15 +129,15 @@ export function ExchangeConfirmationCard(props: Props) {
       <View style={styles.feeContainer}>
         <LineItemRow
           currencySymbol={takerToken}
-          amount={fee}
+          amount={fee || '0.00'}
```

Comment:
Thanks for flagging this- the ExchangeConfirmationCard is now only used for reviewing past exchange (so we should not have to wait to fetch the fee), but the current fee and exchangeRate are being fetched instead of using the historic ones. Have created an issue here to track: https://github.com/celo-org/celo-monorepo/issues/2269